### PR TITLE
Downgraded the package to php7.2 and added contains to AddressableInterface

### PR DESCRIPTION
**Pull request by noahlvb**

### CODE REVIEW

File: src/AddressableInterface.php [hunk 1]
```diff
@@ -12,4 +12,6 @@ interface AddressableInterface
     public function address(): AddressInterface;
 
     public function subnet(): SubnetInterface;
+
+    public function contains(AddressableInterface $address): bool;
```
Comment (noahlvb) [inline src/AddressableInterface.php:16]: Added so that you can check against a Address or network without knowing

**Comment by blackshadev** [comment]
The tests on 7.4 and 7.3 fail due to new mutants, please write tests for to kill them. On PHP 8 it errors completely, how.. cruel it may sound, I rather support PHP 8 than 7.2.

**Comment by noahlvb** [comment]
Yep these downgrades are the newest versions that have ^7.2. I will remove php8 from the matrix then and add more tests

File: src/AddressableInterface.php (hunk 1 above)
Comment (blackshadev) [reply src/AddressableInterface.php:16]: I don't see anything wrong with it. But I don't understand why you would need an Address::contains would be usefull. But for consistency I can get behind it.
Do you have an use-case: When do have an AddressableInterface and not an instance  of Network?

**Comment by blackshadev** [comment]
> Yep these downgrades are the newest versions that have ^7.2. I will remove php8 from the matrix then and add more tests

I am not sure how I feel about this, when will we re-add the PHP 8 support? I rather support 8 than 7.2 since PHP 8 has a feature, 7.2 does not.

**Comment by noahlvb** [comment]
Will drop the PR cause php80 and 72 don't go together.

**Pull request closed without merging**